gpkit/keydict.py

```diff
@@ -44,6 +45,8 @@ def __init__(self, *args, **kwargs):
     def update(self, *args, **kwargs):
         "Iterates through the dictionary created by args and kwargs"
         for k, v in dict(*args, **kwargs).items():
+            if hasattr(v, "copy"):
+                v = v.copy()
```

Comment:
this seems odd and non-dict-like. Why not let whoever is calling `update` copy their own values if they want/need to?